### DISCUSSION TITLE
Support inline @return for method descriptions in Scaladoc

### DISCUSSION THREAD

**Discussion by mbovel** [scala/scala3#18549] [Feature Requests]
https://docs.scala-lang.org/scala3/guides/scaladoc/docstrings.html states:

> don’t repeat yourself. Resist duplicating the method description in the `@return` tag and other forms of repetitive commenting.

But there is currently no technic to reuse the function description in the return description.

Javadoc has a solution for this:

> With javadoc 16 you may make use of the new combo `{@return ...}` tag in order "to avoid duplication of return information in simple situations":
>
> ```java
> /**
> * {@return the name of the object}
> */
> public String getName();
> ```
> 
> Is equivalent to the (still supported) style:
> 
> ```java
> /**
>  * Returns the name of the object.
>  *
>  * @return the name of the object
>  */
> public String getName();
> ```
>
> Find more details at https://bugs.openjdk.java.net/browse/JDK-8075778
> 
> — https://stackoverflow.com/a/66220601

Should we support this in Scaladoc?

**Comment by som-snytt** [comment]
To say something out of scope, I have hoped for the doc tool to support analyzing natural language code comments, so that
```
// the name of the thing
var name: String
```
suffices. A thesis in this area would benefit generations upon generations.

The highly formalized Javadoc style is clearly a dead form.
